drafts/use-cases/5.creating-new-event.md

```diff
@@ -22,25 +23,26 @@ var client = new HydraClient();
 var operation = client.get("http://example.com")
     .getApiDocumentation()
     .getEntryPoint()
-    .getOperations()
-    .filter(op => op.method == "POST" && op.expects == "Event")[0];
+    .onLink('#events')
```

Comment:
I'm not sure what _onLink('#events')_ means.
As for your question, it is still a pseudo code, thus I didn't focus on async or non async conventions - it didn't matter here.
I also woudln't use appendices like _Async_in methods i.e. _getApiDocumentationAsync_. Nowadays it is natural to use async methods, thus sync methods would get such an appendix.
I also didn't want to imply any convention for async methods (i.e. promises, async keyword or callbacks) - it didn't matter here as well. In my latest implementation I started I'm using an async keyword.